packages/auth/src/platform_browser/recaptcha/recaptcha.test.ts

```diff
@@ -46,7 +46,14 @@ describe('platform_browser/recaptcha/recaptcha', () => {
   const GET_RECAPTCHA_CONFIG_RESPONSE: GetRecaptchaConfigResponse = {
     recaptchaKey: 'projects/testproj/keys/' + TEST_SITE_KEY,
     recaptchaEnforcementState: [
-      { provider: 'EMAIL_PASSWORD_PROVIDER', enforcementState: 'ENFORCE' }
+      {
+        provider: RecaptchaProvider.EMAIL_PASSWORD_PROVIDER,
+        enforcementState: EnforcementState.ENFORCE
+      },
+      {
+        provider: RecaptchaProvider.PHONE_PROVIDER,
+        enforcementState: EnforcementState.AUDIT
```

Comment:
Add a combination of `ENFORCE` and `OFF`?

Reply:
Done.